src/main/java/braintrain/model/session/Session.java

```
@@ -0,0 +1,78 @@
+package braintrain.model.session;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import braintrain.model.card.SrsCard;
+import braintrain.model.card.exceptions.MissingCoreException;
+import braintrain.quiz.Quiz;
+import braintrain.quiz.QuizCard;
+
+
+/**
+ * Represents a session that stores cards based on srs data.
+ */
+public class Session {
+    private static final int CARD_COUNT_MINIMUM = 5;
+
```

This does not match the description in `UserGuide.adoc`, do remember to update the `UserGuide.adoc` to match this eventually. Also, could consider changing the access modifier to `public` so it can be used by other classes and given that it is not sensitive info.

This will come in useful when you write a `SessionTest` class which should test whether this restriction is applied: You could pass in `CARD_COUNT_MINIMUM--` of `srsCards` into the `Session` constructor and it should throw an Exception.